docs/starter-web.md

```diff
@@ -0,0 +1,586 @@
+
+??? info "Default configuration set by this library"
+    ```properties
+    --8<-- "sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/defaults.properties"
+
+    --8<-- "sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/monitoring.properties"
+
+    --8<-- "sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/tracing.properties"
+
+    --8<-- "sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/client/default.properties"
+
+    --8<-- "sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/docs/default.properties"
+    ```
+
+**Please make sure to configure `spring.application.name` for every service.**
+
+
+## Authentication
+
+- [Spring Security Documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#web.security)
+
+Enables feature that make a Spring Boot service compliant with the SDA SE Authentication concepts
+using OIDC.
+
+OIDC Authentication can be configured with `auth.issuers` to provide a comma separated
+list of trusted issuers. In develop and test environments, the boolean `auth.disable` may
+be used to disable authentication.
+
+The JWKS URI of each issuer is updated when an unknown Key ID is received and every 5 minutes. The
+cache of known JWK is invalidated after 15 minutes.
+
+!!! warning
+    This setup allows authenticated and anonymous requests! It is the responsibility of policies
+    provided by the Open Policy Agent to decide about denying anonymous requests.
+
+Spring Security is disabled for the Management/Admin Port (default: 8081). Be aware that these port
+should not be accessible out of the deployment context.
+
+This security implementation lacks some features compared to [sda-dropwizard-commons](https://github.com/SDA-SE/sda-dropwizard-commons/tree/master/sda-commons-server-auth):
+- No configuration of static local public keys to verify the token signature. 
+- No configuration of JWKS URIs to verify the token signature. 
+- The IDP must provide an `iss` claim that matches the base URI for discovery. 
+- Leeway is not configurable yet. 
+- The client that loads the JWKS is not configurable yet.
+
+## Authorization
+
+Enables feature that make a Spring Boot service compliant with the SDA SE Authorization concepts
+using Open Policy Agent.
+
+The authorization is done by the [Open Policy Agent](https://www.openpolicyagent.org/). It can be 
+configured with [`opa.` configuration properties](#configuration).
+
+??? info "Requests to the Open Policy Agent"
+    Requests to the server are determined by the base URL and the policy package.
+    Given the default base URL `http://localhost:8181` and an example package of `com.my.service`,
+    the Open Policy Agent server will be requested for authorization decision at
+    `http://localhost:8181/v1/data/com/my/package`.
```

Comment:
Shouldn't it be `http://localhost:8181/v1/data/com/my/service` then?

Reply:
fixed